docs/Introduction/tutorials/beginners-tutorial.md

```diff
@@ -168,25 +149,13 @@ Before using data feeds, it's important to understand how interfaces work in Sol
 Interfaces define functions without their implementation, leaving inheriting contracts to define the actual implementation themselves. This makes it easier to know what functions to call in a contract. Here's an example of an interface:
 
 ```solidity
-pragma solidity 0.8.7;
-
-interface numberComparison {
-   function isSameNum(uint a, uint b) external view returns(bool);
-}
-
-contract Test is numberComparison {
-    
-   constructor() {}
-   
-   function isSameNum(uint a, uint b) external view returns(bool){
-      if (a == b) {
-        return true;
-      } else {
-        return false;
-      }
-   }
-}
+{% include samples/Tutorials/Test.sol %}
 ```
+<div class="remix-callout">
+  <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/Tutorials/Test.sol" target="_blank" class="cl-button--ghost">Deploy this contract using Remix ↗</a>
+</div>
+
+For this example, `override` is necessary to use in the `Test` contract function since it is overriding the base function contained in the `numberComparison` interface. `pure` is also used in this example: since the `isSameNum` function in the `Test` contract does not return a storage variable, the contract uses `pure` instead of `view`.  
```

Comment:
```suggestion
For this example, `override` is necessary in the `Test` contract function because it overrides the base function contained in the `numberComparison` interface. The contract uses `pure` instead of `view` because the `isSameNum` function in the `Test` contract does not return a storage variable. 
```